docs/plan/2026-04-15-review-fix-step4-plan.md

```
@@ -0,0 +1,27 @@
+# review-fix step4 計画
+
+## 目的
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)